app-dashboard/js/view/panel-dashboard.js

```diff
@@ -48,13 +48,21 @@ define([
             let flood_acquisition_date = new Date(floodCollectionView.selected_forecast.attributes.acquisition_date);
             let flood_forecast_date = new Date(floodCollectionView.selected_forecast.attributes.forecast_date);
 
+            let lead_time = flood_forecast_date.getDate() - new Date().getDate();
+            let event_status = 'Current';
+            if(lead_time < 0){
+                lead_time = '-';
```

Comment:
It was negative because you count the acquisition date from today.
Forecast event will always have lead_time.
To see if it is historical or not, we will put the attributes/function that evaluates this in the model.
I think that is the better way.

Reply:
use
```
floodCollectionView.selected_forecast.attributes.is_historical
```

To determine if it's historical or not.
Even if it's historical, it will have lead time, so just show the lead time as it is